### PR TITLE
Split Layout to make code DRY

### LINKED ISSUE #16: Split Layout files
Split Header, Footer in separate layout files.

*Working on it*

### PR DESCRIPTION
**Pull request by niteshkumarniranjan**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes #16 
- Preview Link https://niteshkumarniranjan.github.io/gci18.fossasia.org/

## Description / Changes
Splitted Header and Footer into separate files to make it DRY.

screenshot from 2018-10-25 19-08-18

### CODE REVIEW

**Comment by niteshkumarniranjan** [comment]
@fragm3 I have just removed a class thats why this is and about the logo I think this logo looks better in a single line rather than having two line logo.

**Comment by niteshkumarniranjan** [comment]
@fragm3 I have been doing UI/UX designs for more than a year and I think that having a light background adds contrasts so that people see the Banner first not the header.

**Comment by niteshkumarniranjan** [comment]
@fragm3 Reverted back to grey menu.

**Comment by fragm3** [comment]
@niteshkumarniranjan one PR for one issue. Make it exactly same, you can create new enhancement issues for improvements. Helps us keep track of work you have done, quick code review.

**Comment by niteshkumarniranjan** [comment]
@fragm3 I have reverted back to gray menu so there is only one issue being addressed.

**Comment by saarthakchats** [comment]
Don't repeat yourself @YashKumarVerma

**Comment by niteshkumarniranjan** [comment]
@saarthakchats that favivon isn't coming from my PR

File: _includes/header.html [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+  <link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all" />
+  <link href="css/themify-icons.css" rel="stylesheet" type="text/css" media="all" />
+  <link href="css/flexslider.css" rel="stylesheet" type="text/css" media="all" />
+  <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all" />
+  <link href="css/ytplayer.css" rel="stylesheet" type="text/css" media="all" />
+  <link href="css/theme.css" rel="stylesheet" type="text/css" media="all" />
+  <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
+  <link href='http://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600'
+    rel='stylesheet' type='text/css'>
+</head>
```
Comment (niteshkumarniranjan) [inline _includes/header.html]: @saarthakchats Check the code there is no favicon linked.

**Comment by saarthakchats** [comment]
Okay cool

**Comment by saarthakchats** [comment]
@YashKumarVerma 
DRY means Don't Repeat Yourself the main aim of which is to reduce repetition of code.

**Comment by niteshkumarniranjan** [comment]
@saarthakchats I cann't if I will squash now conflict will be back.

**Comment by tabesin** [comment]
Why do you delete index html?

**Comment by niteshkumarniranjan** [comment]
@niteshkumarniranjan I haven't deleted index.html

File: _includes/footer.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    </div>
+    <!--end of row-->
+    <div class="row">
+      <div class="col-sm-6">
+        <span class="sub">&copy; Copyright 2016 - Medium Rare</span>
+      </div>
+      <div class="col-sm-6 text-right">
+        <ul class="list-inline social-list">
+          <li>
+            <a href="#">
```
Comment (STEdirisuriya) [inline _includes/footer.html]: Add links there.

**Comment by realslimshanky** [comment]
@niteshkumarniranjan can you confirm this pr also closes https://github.com/fossasia/gci18.fossasia.org/issues/27 ?

**Comment by niteshkumarniranjan** [comment]
@realslimshanky Nope! That's another issue.

**Pull request merged**